Assets/MixedRealityToolkit.Tests/PlayModeTests/PointerBehaviorTests.cs

```diff
@@ -0,0 +1,283 @@
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.Input;
+using System;
+using System.Collections.Generic;
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    /// <summary>
+    /// Verify that pointers can be turned on and offvia FocusProvider.SetPointerBehavior
+    /// </summary>
+    public class PointerBehaviorTests : BasePlayModeTests
+    {
+        private class PointerStateContainer
+        {
+            public bool? LinePointerEnabled { get; set; }
+            public bool? SpherePointerEnabled { get; set; }
+            public bool? PokePointerEnabled { get; set; }
+            public bool? GazePointerEnabled { get; set; }
+            public bool? GGVPointerEnabled { get; set; }
+        }
+
+        private void EnsurePointerStates(Handedness h, PointerStateContainer c)
+        {
+            Action<IMixedRealityPointer, string, bool?> helper = (ptr, name, expected) =>
+            {
+                if (!expected.HasValue)
+                {
+                    Assert.Null(ptr, $"Expected {h} {name} to be null but it was not null");
+                }
+                else
+                {
+                    Assert.NotNull(ptr, $"Expected {name} to not be null, but it was null");
+                    Assert.AreEqual(expected.Value, ptr.IsInteractionEnabled,
+                    $"Incorrect state for {h} {name}.IsInteractionEnabled");
+                }
+
+            };
+            helper(PointerUtils.GetPointer<LinePointer>(h), "Line Pointer", c.LinePointerEnabled);
+            helper(PointerUtils.GetPointer<SpherePointer>(h), "Sphere Pointer", c.SpherePointerEnabled);
+            helper(PointerUtils.GetPointer<PokePointer>(h), "Poke Pointer", c.PokePointerEnabled);
+            helper(PointerUtils.GetPointer<GGVPointer>(h), "GGV Pointer", c.GGVPointerEnabled);
+            helper(CoreServices.InputSystem.GazeProvider.GazePointer, "Gaze Pointer", c.GazePointerEnabled);
+        }
+
+        /// <summary>
+        /// Tests that the gaze pointer can be turned on and off
+        /// </summary>
+        /// <returns></returns>
+        [UnityTest]
+        public IEnumerator TestGaze()
+        {
+            PointerStateContainer gazeOn = new PointerStateContainer()
+            {
+                GazePointerEnabled = true,
+                GGVPointerEnabled = true,
+                PokePointerEnabled = null,
+                SpherePointerEnabled = null,
+                LinePointerEnabled = null
+            };
+
+            // set input simulation mode to GGV
+            PlayModeTestUtilities.SetHandSimulationMode(HandSimulationMode.Gestures);
+
+            TestHand rightHand = new TestHand(Handedness.Right);
+            TestHand leftHand = new TestHand(Handedness.Left);
+
+            yield return rightHand.Show(Vector3.zero);
+            yield return leftHand.Show(Vector3.zero);
+
+            TestContext.Out.WriteLine("Show both hands");
```

Comment:
>TestContext.Out.WriteLine("Show both hands"); [](start = 12, length = 45)

Nit: I think this WriteLine should be a few lines above (when it is showing the hands).